tests/test_pandas.py

```diff
@@ -131,3 +131,13 @@ def test_datetime_index_serialize():
         df = pd.read_csv(tmpdir / "test.csv", names=["date", "c1"], index_col="date", header=0)
 
     df._ipython_display_()
+
+
+def test_cut_mixed_types():
```

Comment:
See also [this test](https://github.com/lux-org/lux/blob/master/tests/test_pandas_coverage.py#L669).